hbase_storage/src/main/java/com/splicemachine/timestamp/hbase/ZkTimestampSource.java

```diff
@@ -132,4 +132,14 @@ public synchronized void shutdown() {
             _tc = null;
         }
     }
+
+    @Override
+    public void refresh() {
```

Comment:
I don't like the refresh() approach, we should take care of that when we fail over

Reply:
We need to advance timestamp whenever replica has replicated all WAL edits of a snapshot.

Reply:
I thought we are advancing the timestamp directly in ZK and then calling this method to refresh the Timestamp server, however we don't really need to update the Timestamp server since we don't create new transactions, do we?

Reply:
We should allow creating read-only transactions on replica database, because it may be used to serve read requests. At least the DBA may want to run some queries on replica to trouble-shooting replication, check its progress, etc.